docs/src/documentation/03-components/03-information/toast/03-accessibility.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/toast/accessibility/
+---
+
+## Accessibility
+
+The Toast component has been designed with accessibility in mind, providing a non-intrusive way to display temporary notifications to users. It follows accessibility best practices to ensure all users, including those using assistive technologies, can perceive and understand these notifications.
+
+### Accessibility props
+
+| Name      | Type                               | Default    | Description                                                                                             |
+| :-------- | :--------------------------------- | :--------- | :------------------------------------------------------------------------------------------------------ |
+| role      | `"status" \| "alert"`              | `"status"` | The ARIA role for the toast. Use `"alert"` for important notifications requiring immediate attention    |
+| aria-live | `"polite" \| "assertive" \| "off"` | `"polite"` | Controls how urgently screen readers announce the content. Use `"assertive"` only for critical messages |
+
+These attributes are passed through the `ariaProps` object in the second argument of the `createToast` function:
+
+```jsx
+createToast("Your message", {
+  ariaProps: {
+    role: "alert",
+    "aria-live": "assertive",
+  },
+});
+```
+
+### Automatic accessibility features
+
+- **Focus management**: Toasts do not steal focus from the user's current task, maintaining keyboard navigation flow
+- **Animation pausing**: Hover over toasts pauses the dismissal countdown, giving users more time to read lengthy messages
+
+### Keyboard and screen reader experience
+
+- **Visual notifications**: Toasts appear visually and are automatically announced to screen readers
+- **Keyboard focus**: Toasts don't steal keyboard focus, allowing users to continue their current task
+- **Mouse hover**: Pauses the toast dismissal timer
+- **Mouse leave**: Resumes the toast dismissal timer
+- **Interactive elements**: Any interactive elements within toasts are part of the natural tab order
```

Comment:
Is this true? How does it actually work for the user if the focus stays where it was?